03-algorithms-sorting/quick_sort.rb

```diff
@@ -0,0 +1,40 @@
+class QuickSort
```

Comment:
- [ ] This looks great, the only thing I would suggest is that since you're always passing `0` and `array.length - 1` as the parameters to `quick_sort` for every case, you can probably omit them and figure them out dynamically within `quick_sort`. In other words, you should only need to pass in `array` into `quick_sort` as a parameter.